edge/build.json

```diff
@@ -1,9 +1,9 @@
 {
-    "build_from": {
-      "aarch64": "homeassistant/aarch64-base-python:3.9-alpine3.13",
-      "amd64": "homeassistant/amd64-base-python:3.9-alpine3.13",
-      "armhf": "homeassistant/armhf-base-python:3.9-alpine3.13",
-      "armv7": "homeassistant/armv7-base-python:3.9-alpine3.13",
-      "i386": "homeassistant/i386-base-python:3.9-alpine3.13"
-    }
+  "build_from": {
+    "aarch64": "homeassistant/aarch64-base-python:3.10-alpine3.18",
```

Comment:
what is the main purpose for updating image of addon?

Reply:
#165 I prefer to make small changes, so I wanted to bump just the OS version before making the s6-overlay changes, I'm still working on it.